test/stablex/stablecoin_converter.js

```diff
@@ -40,6 +39,19 @@ const {
 
 const feeDenominator = 1000 // fee is (1 / feeDenominator)
 
+const fiveThousand = new BN("5000")
+const tenThousand = new BN("10000")
+const genericDepositsAndOrders = {
```

Comment:
Is this really generic? Isn't it too small in the first place?

Reply:
ya I suppose so. Got a name suggestion?